src/storage.rs

```diff
@@ -59,11 +81,29 @@ pub struct AccountBasic {
 }
 
 impl AccountBasic {
+    /// Create a new account with the given balance, nonce, code hash and code.
```

Comment:
The `struct` is pretty simple with all `pub` fields so constructing with `AccountBasic {` should be good enough for now.